packages/base/src/signals/signal-manager.ts

```diff
@@ -4,17 +4,21 @@ import { Experiment } from 'tsp-typescript-client/lib/models/experiment';
 import { Trace } from 'tsp-typescript-client/lib/models/trace';
 import { OpenedTracesUpdatedSignalPayload } from './opened-traces-updated-signal-payload';
 import { OutputAddedSignalPayload } from './output-added-signal-payload';
+import { TimeRangeUpdatePayload } from './time-range-data-signal-payloads';
+
 export declare interface SignalManager {
     fireTraceOpenedSignal(trace: Trace): void;
     fireTraceDeletedSignal(trace: Trace): void;
     fireExperimentExperimentSignal(experiment: Experiment): void;
     fireExperimentClosedSignal(experiment: Experiment): void;
     fireExperimentDeletedSignal(experiment: Experiment): void;
     fireExperimentSelectedSignal(experiment: Experiment | undefined): void;
+    fireExperimentUpdatedSignal(experiment: Experiment): void;
     fireOpenedTracesChangedSignal(payload: OpenedTracesUpdatedSignalPayload): void;
     fireOutputAddedSignal(payload: OutputAddedSignalPayload): void;
     fireItemPropertiesSignalUpdated(properties?: { [key: string]: string }): void;
     fireThemeChangedSignal(theme: string): void;
+    // TODO - Refactor or remove this signal.  Similar signal to fireRequestSelectionRangeChange
```

Comment:
fireSelectionChangedSignal includes other data in the payload, receiving components can synchronize on. I agree that they are similar and should be refactored.

Reply:
Lets' keep that for later!